www/src/pages/foundations/colors.jsx

```diff
@@ -0,0 +1,320 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import classNames from 'classnames';
+import SEO from '../../components/seo';
+
+const utilityClasses = {
+  bg: (color, level) => (level ? `bg-${color}-${level}` : `bg-${color}`),
+  border: (color, level) => (level ? `border-${color}-${level}` : `border-${color}`),
+  text: (color, level) => (level ? `text-${color}-${level}` : `text-${color}`),
+};
+
+const colors = [
+  { themeName: 'gray', color: 'gray', unusedLevels: [] },
+  { themeName: 'primary', color: 'blue', unusedLevels: [] },
+  { themeName: 'success', color: 'green', unusedLevels: [] },
+  { themeName: 'info', color: 'teal', unusedLevels: [] },
+  { themeName: 'danger', color: 'red', unusedLevels: [] },
+  { themeName: 'warning', color: 'yellow', unusedLevels: [500, 700, 900] },
+];
+
+const levels = [
+  { level: 100, use: 'Backgrounds' },
+  { level: 200, use: 'Borders & Lines' },
+  { level: 300, use: 'Icons & Semantic State' },
+  { level: 500, use: 'Lighter Text & Element Fills' },
+  { level: 700, use: 'Regular Text & Hover State' },
+  { level: 900, use: 'Darker Text & Active State' },
+];
+
+const selectorColors = {};
+
+function parseColors(cssSelectors) {
+  const colorsAreParsed = Object.keys(selectorColors).length !== 0;
+  if (colorsAreParsed) return;
+
+  cssSelectors.forEach(({ selector, declarations }) => {
+    // All delcarations fit this shape: "background-color: #fff !important;"
+    const declarationFragments = declarations[0].split(' ');
```

Comment:
Seems like this and the right-hand side of the assignment below don't need to be in the forEach?